lexicons/tools/ozone/moderation/defs.json

```diff
@@ -386,6 +392,45 @@
         }
       }
     },
+    "accountEvent": {
+      "type": "object",
+      "description": "Logs account status related events on a repo subject. Normally captured by automod from the firehose and emitted to ozone for historical tracking.",
+      "properties": {
+        "comment": { "type": "string" },
+        "accountStatus": {
+          "type": "string",
+          "knownValues": ["deactivated", "activated", "deleted"]
+        },
+        "timestamp": {
+          "type": "string",
+          "format": "datetime"
+        }
+      }
+    },
+    "identityEvent": {
```

Comment:
for completeness, there is a concept of "tombstoning" an identity. this has been very rare in practice. in theory it is a permanent deletion, though technically it can be reverted in PLC within a time period (72hr), and can be reverted any time with `did:web`.

it is sort of a corner-case, but would actually be a good thing to track in ozone because otherwise it would be very hard for mods to understand what had happened: a deleted identity makes it pretty hard to learn anything about an account from the live network.